data/changelogs/workers.yaml

```
@@ -4,6 +4,12 @@ productName: Workers
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+- publish_date: '2024-05-17'
+  description: |-
+    - Update v8 to 12.6
+    - DigestStream is now more efficient and exposes a new bytesWritten property that indicates that number of bytes written to the digest.
+    - The new `fetch_standard_url` compatibility flag will become active by default on June 3rd, 2024 and ensures that URLs passed into the fetch(...) API, the new Request(...) constructor, and redirected requests will be parsed using the standard WHATWG URL parser.
+    - A bug in the fetch API implementation would cause the content type of a Blob to be incorrectly set. Unfortunately, the fix for this bug potentially breaks some existing code so the fix is being released behind a new `blob_standard_mime_type` compatibility flag.
```

```suggestion
    - Adds the `blob_standard_mime_type` compatibility flag, which fixes a bug in the Workers runtime's implementation of the fetch API that caused the content type of a Blob to be incorrectly set.
```

Link to docs for the compat flag? When is it default enabled?